ui/js/services/jobfilters.js

```diff
@@ -477,7 +477,7 @@ treeherder.factory('thJobFilters', [
         }
 
         function _stripClearableFieldFilters(locationSearch) {
-            _.forEach(locationSearch, function (val, field) {
+            Object.keys(locationSearch, function (field) {
```

Comment:
I think this should have a ``forEach`` like the first change.  Would you also convert these to ``=>`` functions while you're here?  Thanks!